cmd/kpod/docker/types.go

```diff
@@ -0,0 +1,271 @@
+package docker
```

Comment:
Pulling in the upstream definitions for some of these types also pulls in dependencies of the packages they're in, which previously led to us pulling in dockerd's copies of its graph drivers, creating duplicate-symbol-definition errors with some of the cgo code that we inherited in the storage library.

The references to the definitions come from multiple repositories, some of which aren't under the moby account, but I'm only attached to the package name to the extent that it's part of the API.